### PR TITLE
fix: prevent genbank ingest log spam

### PR DESCRIPTION
**Pull request by ivan-aksamentov**
Resolves https://github.com/nextstrain/ncov-ingest/issues/250

The stdout of GenBank transform is currently printed to stdout of AWS Batch. This PR redirects it to a file. The file is then uploaded to Slack. This is similar to how it's handled for GISAID.

This facilitates reading the logs for techies and makes it more convenient to get domain-specific output for scientists.

The code is a copypasta from the similar GISAID code. I hope there are no typos :)

### CODE REVIEW

**Comment by emmahodcroft** [comment]
This sounds like a great idea Ivan. I'm not best to review the code in detail, but the idea is good. 

@MoiraZuber important that this sounds like it'll work for you too! If you have any questions or want to clarify with Ivan about how this works, don't hesitate to ask!

**Review by victorlin — APPROVED** [review]

**Review by joverlee521 — APPROVED** [review]
Looks good from inspecting the code. I didn't find any typos 👍

File: Snakefile [hunk 1]
```diff
@@ -129,14 +129,15 @@ rule transform_genbank_data:
...
+        flagged_annotations = temp("data/genbank/flagged-annotations"),
         duplicate_biosample = "data/genbank/duplicate_biosample.txt"
     shell:
         """
         ./bin/transform-genbank {input.ndjson} \
             --biosample {input.biosample} \
             --duplicate-biosample {output.duplicate_biosample} \
             --output-metadata {output.metadata} \
-            --output-fasta {output.fasta}
+            --output-fasta {output.fasta} > {output.flagged_annotations}
```
Comment (corneliusroemer) [inline Snakefile:140]: Should we redirect stderr too?

Or even `tee` it so it's visible in Slack and in the logs? That might be most useful. We want to know if there are errs.

Comment (ivan-aksamentov) [reply Snakefile:140]: @corneliusroemer The goal of the PR is to add that 1 statement that was forgotten when copied from gisaid script, ant to allow engineers to read logs.

I'd avoid messing with stderr here, but we may consider it if it is needed in a separate PR. In that case we'd probably better tp upload error logs for all stages and all databases, not just this one thing. Transform stage is actually somewhat stable and unlikely to emit errors.

**Pull request merged**